app/org/sagebionetworks/bridge/hibernate/HibernateAccount.java

```diff
@@ -250,7 +267,18 @@ public AccountStatus getStatus() {
     public void setStatus(AccountStatus status) {
         this.status = status;
     }
-
+    
+    /** @see #getClientData */
+    @Column(columnDefinition = "mediumtext", name = "clientData", nullable = true)
+    public String getClientData() {
```

Comment:
Is this a text or a mediumtext? The wiki says text, but the pull request and the annotations say mediumtext.

Reply:
Updated bootstrap document (it's mediumtext)